docs/transaction.md

```diff
@@ -2,17 +2,11 @@
 
 ## UTXO overview
 
-**TODO maybe system/model instead of structure?**
-Mintlayer uses the Bitcoin UTXO structure instead of the account-based models of Ethereum, Ripple, Stellar, and others.
-**TODO need explanation of this sentence, it is not clear to me**
-Since each transaction's output is stored separately (even when sent to a single address), it is only possible to spend the entire transaction’s output.
+Mintlayer uses the Bitcoin UTXO system instead of the account-based models of Ethereum, Ripple, Stellar, and others. There are three essential reasons for this: 
 
-There are three essential reasons for choosing the UTXO model: 
-
-- It is compatible with technologies already implemented in Bitcoin, such atomic swaps and the Lightning Network.
+- The utxo model is compatible with technologies already implemented in Bitcoin, such atomic swaps and the Lightning Network.
  
-**TODO - why does this improve privacy?**
-- It is more privacy-oriented: a single wallet usually utilizes multiple addresses, making it difficult and sometimes impossible to determine which addresses belong to whichs user. 
+- The utxo model is more privacy-oriented: a single wallet usually utilizes multiple addresses, making it difficult and sometimes impossible to determine which addresses belong to which user.
```

Comment:
I'd say `it's possible to utilise multiple addresses` rather than `usually utilises` while it might be true for a certain chunk of the population I have no idea if that claim holds for the wider world

Reply:
Changed